tests/geomultiscale-mapping/RadialGeoMultiscale.cpp

```diff
@@ -0,0 +1,82 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/Participant.hpp>
+#include <vector>
+#include "testing/TestContext.hpp"
+
+using namespace precice;
+using precice::testing::TestContext;
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(GeomultiscaleMapping)
+BOOST_AUTO_TEST_CASE(RadialGeoMultiscale)
+{
+  PRECICE_TEST("SolverOne"_on(1_rank), "SolverTwo"_on(1_rank));
+  using Eigen::Vector3d;
+
+  Participant cplInterface(context.name, context.config(), 0, 1);
+  if (context.isNamed("SolverOne")) {
+    auto     meshName = "MeshOne";
+    Vector3d vec1     = Vector3d::Constant(0.0);
+    auto     vid      = cplInterface.setMeshVertex(meshName, vec1);
+    auto     dataAID  = "DataOne";
+    auto     dataBID  = "DataTwo";
+
+    double valueDataB = 1.0;
```

Comment:
I guess this should not be initialized with the target value, right? Because if I remove the `readData`, the test would still pass.

Reply:
You're right. sorry that was oversight on my part!